docs/pages/jpscore/jpscore_operativ.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Operational models
+keywords: simulation
+tags: [jpscore, model]
+sidebar: jupedsim_sidebar
+folder: jpscore
+permalink: jpscore_operativ.html
+summary: Several operational models are implemented in jpscore. An operational model defines how pedestrians moves from one time step to the next.
+last_updated: Dec 21, 2019
+---
+
+## Introduction
+In the definition of agent's properties it is mandatory to precise the number of the model to be used e.g.:
+
+```xml
+ <agents operational_model_id="n">
+```
+The definition of any model parameter is composed of two different
+sections:
+
+- **model_parameters**: Model specific parameter. See below in the different model sections.
+- **agent_parameters**: These parameter are mainly specific for the shape of pedestrians
+    or other pedestrian properties like desired speed, reaction time etc.
+
+## Model parameters (in general)
+- `<solver>euler</solver>`
+     - The solver for the ODE. Only *Euler*. No other options.
+- `<stepsize>0.001</stepsize>`:
+     - The time step for the solver. This should be choosed with care. For force-based model it is recommended to take a value between $$ 10^{-2} $$ and $$10^{-3}$$ s.
+       For first-order models, a value of 0.05 s should be OK.
+       A larger time step leads to faster simulations, however it is too risky and can lead to
+numerical instabilities, collisions and overlapping among pedestrians.
+     - Unit: s
+- `<periodic>0</periodic>`
+     - Set to 1 if a system with closed boundary conditions should be simulated. Default setting is 0.
+     - This option is only implemented in *Tordeux2015* and is very geometry-specific (only for corridors) with predefined settings. 
+
```

Comment:
Can we just remove this part, if it is only usable in a few special occasions?

Reply:
Yes, this is very hacky. I'll remove is.